runtime/compiler/env/CHTable.cpp

```diff
@@ -1015,6 +1131,16 @@ TR_ClassQueries::collectImplementorsCapped(TR_PersistentClassInfo *clazz,
    {
```

Comment:
I just saw that parameter called `locked`. If this is executed in a critical section we may not be able to send a message because of the delay.